ios/Podfile.lock

```diff
@@ -986,4 +986,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: bb6529ab88d2e95007f12b67f3974e0e60c964a6
 
-COCOAPODS: 1.12.1
+COCOAPODS: 1.12.0
```

Comment:
I did a `brew update` which updated the cocoapods so I not have a discrepancy with the one running on circleci, fyi